src/models/sqlOutputContentProvider.ts

```diff
@@ -75,7 +75,10 @@ export class SqlOutputContentProvider {
     }
 
     public editorSelectionRequestHandler(uri: string, selection: ISelectionData): void {
-        this._queryResultsMap.get(uri).queryRunner.setEditorSelection(selection);
+        const docExists = this._vscodeWrapper.textDocuments.find(doc => doc.uri.toString() === uri);
```

Comment:
Does it make more sense to move this check into the query runner, since that is where the document is actually attempted to open?